portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Resources/components/operationComponents/OperationGovernance.jsx

```
@@ -56,19 +56,28 @@ export default function OperationGovernance(props) {
+        return null;
+    }
+    
     // Get operation scopes - handle MCP servers differently since they store scopes directly
     const operationScopes = isMCPServer && operation.scopes 
         ? operation.scopes 
         : getOperationScopes(operation, spec);
     const isOperationRateLimiting = api.apiThrottlingPolicy === null || 
         !componentValidator.includes('apiLevelRateLimiting');
     const filteredApiScopes = api.scopes.filter((sharedScope) => !sharedScope.shared);
-
     const intl = useIntl();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)